### PR TITLE
Gh429 missing entrypoint

### PR DESCRIPTION
**Pull request by chrisforbes**

### CODE REVIEW

**Comment by MarkMLunarG** [comment]
LGTM thanks for sorting this out.

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -2744,6 +2743,7 @@ static bool validate_pipeline_shader_stage(debug_report_data *report_data,
     auto pipelineLayout = pipeline->pipeline_layout;
 
     /* validate push constant usage */
```
Comment (tobine) [inline layers/core_validation.cpp]: May want to alter comment

File: layers/core_validation.cpp [hunk 2]
```diff
@@ -2855,6 +2855,11 @@ static bool validate_and_capture_pipeline_shader_state(debug_report_data *report
                                                enabledFeatures, shaderModuleMap);
     }
 
+    // if the shader stages are no good individually, cross-stage validation is
+    // pointless.
```
Comment (tobine) [inline layers/core_validation.cpp]: clang-format? Seems like this could all fit on one line

**Comment by tobine** [comment]
Just super-minor comment comments
LGTM :+1:

**Comment by chrisforbes** [comment]
On combining/splitting tests, I'd strongly prefer separate tests wherever it's practical.

**Pull request merged**